docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.1"
+
+services:
+  nextcloud:
+    image: nextcloud/all-in-one
+    restart: unless-stopped
+    container_name: nextcloud-aio
+    volumes:
+      - ./config:/mnt/docker-aio-config
```

Comment:
This actually needs to be inside the nextcloud_aio_mastercontainer volume. otherwise the built-in borgbackup will not work.

Reply:
```suggestion
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
```

Reply:
how does the resulting volume name look like if you put in this volume name?
could you please output a docker volume ls here? Thanks!

Reply:
What do you mean exactly?

Reply:
the built-in backup cannot backup the important files that are usually stored in the docker volume if you don't store it in a volume ;)

Reply:
see https://github.com/nextcloud/all-in-one/blob/02365fdd3f91decdea6da174697fe18eedc13647/php/containers.json#L230-L234

Reply:
> how does the resulting volume name look like if you put in this volume name?
could you please output a docker volume ls here?

If we define the volume as external we can make sure it has the exact name we need:
```
volumes:
  nextcloud_aio_mastercontainer:
    external: true
```

Reply:
> 

nice, thanks for the tip!

Reply:
@AlexanderProd please add that :)

Reply:
`Volume nextcloud_aio_mastercontainer declared as external, but could not be found. Please create the volume manually using `docker volume create --name=nextcloud_aio_mastercontainer` and try again.` 
This is what I get when starting the container with `external: true`. Do we really want it to be necessary that the user has to create the volume manually?

Reply:
> Do we really want it to be necessary that the user has to create the volume manually?

Not necessarily... But definitely if there is no other way to name it like this.

Reply:
Weird, it works on my machine:

```
ag@kronos ~
$ sudo docker volume ls
DRIVER    VOLUME NAME
ag@kronos ~
$ sudo docker-compose up -d
[+] Running 3/3
 ⠿ Network ag_default                       Create...                      0.0s
 ⠿ Volume "nextcloud_aio_mastercontainer"   Created                        0.0s
 ⠿ Container nextcloud-aio-mastercontainer  Started                        0.3s
ag@kronos ~
$ sudo docker volume ls
DRIVER    VOLUME NAME
local     nextcloud_aio_mastercontainer
ag@kronos ~
```

Reply:
@pestotoast whats the excact compose file that you've used? or what were the changes that you've made to the one in this PR?

Reply:
```
version: "3.1"

volumes:
  nextcloud_aio_mastercontainer:
    external: true

services:
  nextcloud:
    image: nextcloud/all-in-one:latest
    restart: unless-stopped
    container_name: nextcloud-aio-mastercontainer
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
      - /var/run/docker.sock:/var/run/docker.sock:ro
    ports:
      - 80:80
      - 8080:8080
      - 8443:8443

```

Reply:
Well, looks just like mine except I had the volumes defined at the end but putting them at the begging didn't change it either. Shall I just create it manually and move on? 

`
$ docker version
`
Client: Docker Engine - Community
 Version:           20.10.11
 API version:       1.41
 Go version:        go1.16.9
 Git commit:        dea9396
 Built:             Thu Nov 18 00:37:06 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.11
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.9
  Git commit:       847da18
  Built:            Thu Nov 18 00:35:15 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.12
  GitCommit:        7b11cfaabd73bb80907dd23182b9347b4245eb5d
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

Reply:
Does this work for you?
```
volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer
```

Reply:
as I said: if there is another way to provide this excact name for the volume, I am open to add that. But we definitely need this excact name. So if there is no other way, I guess we should just go with that also if it requires to create the volume first...

Reply:
Using `name: nextcloud_aio_mastercontainer` fixed it for me.

Reply:
great 🎉